modules/hcp-aws-deploy-hc.adoc

```diff
@@ -43,6 +44,7 @@ $ hcp create cluster aws \
 <7> Specify the node pool replica count, for example, `3`.
 <8> By default, all `HostedCluster` and `NodePool` custom resources are created in the `clusters` namespace. You can use the `--namespace <namespace>` parameter, to create the `HostedCluster` and `NodePool` custom resources in a specific namespace.
 <9> Specify the Amazon Resource Name (ARN), for example, `arn:aws:iam::820196288204:role/myrole`.
+<10> To specify whether the EC2 instance should run on shared or single-tenant hardware, include this flag.
```

Comment:
maybe explain what the `render` flag does first, e.g. it won't apply the k8s resources, but render them into a file instead. Then you can edit the NodePool spec in that yaml 'To specify whether the EC2 instance should run on shared or single-tenant hardware'